data/json/recipes/recipe_others.json

```diff
@@ -4347,5 +4347,27 @@
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "splinter", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "sandbag",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "time": 2000,
+    "autolearn": true,
+    "qualities": [ { "id": "DIG", "level": 3 } ],
```

Comment:
I'm not sure why digging 3 would be needed, really you can fill sandbags with your hands if you want. Digging 1 would probably be plenty.

Reply:
I can see an argument to be made about what is and is not needed, but `digging 1` is basically "a sharp stick" rather than anything that can actually be used to effectively move sand.

Reply:
Note that the soil or sand used here is already dug up, and probably in a loose pile. Since you have to get the material somehow tool quality is still a factor, but when filling the bag it should not matter that much. Alternative would to make duplicate recipies for different speeds and shovel qualities, but that seems bloated.

Reply:
I've been pushing for the ability to make a second quality field that allows certain qualities to speed up the recipe, but not be required to make it. that would be appropriate here. Really I don't think any tool quality at all is needed to fill a sandbag.

Reply:
A trowel also have digging 1 quality.
At least decrease this to digging level 2, otherwise one wouldn't be able to use makeshift shovels for filling a bag.